andela/titan-dev-ah-frontend
# #165082449 shared components

#### What does this PR do?
This PR creates the basic components that will be reused in almost application's pages
#### Description of Task to be completed?
* Created the buttons
* Created the inputs
* Created the cards
#### How should this be manually tested
1.  After cloning the project repository with `git clone https://github.com/andela/titan-dev-ah-frontend.git`,
2. Run the project dependencies installation with `yarn` or `npm install`
3. Start the React development server using `yarn start` or `npm run start`: You shall see the `Home` page that has all of the created components.
4. To create the transpiled package, you can use the following command: `yarn build` or `npm run build` and everything shall be transipiled and saved into `dist` directory.
Finally, you shall get your application started in the default browser.

#### Any background context you want to provide?
* In the `src/views/Home.js` file, You might notice that they are some inline styles, or any other inappropriate lines of code like: `console.log()`, they were for testing how the reusable components inteact with the component that called them. They shall be removed in the future.
#### What are the relevant pivotal tracker stories?
#165082449 
#### Questions:
None
